.settings/com.google.gdt.eclipse.core.template.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+jarsExcludedFromWebInfLib=*/src/test/resources/lib/appengine/appengine-api-labs.jar|*/src/test/resources/lib/appengine/appengine-api-stubs.jar|*/src/test/resources/lib/appengine/appengine-api.jar|*/src/test/resources/lib/appengine/appengine-remote-api.jar|*/src/test/resources/lib/appengine/appengine-testing.jar|*/src/test/resources/lib/httpunit/httpunit.jar|*/src/test/resources/lib/javamail/mail.jar|*/src/test/resources/lib/selenium/selenium-server-standalone-2.46.0.jar
+warSrcDir=src/main/webapp
+warSrcDirIsOutput=true
```

Comment:
However, the template approach has the downside that all future updates to that file will not reach existing devs, because it will no longer be tracked. We have the same problem for build.properties and test.properties files. Can you split this into two PRs? Fixing problems in the code should be separate from ignoring the jar files. For the latter, there should be some changes to the .ignore file?